tardis/adapters/sites/kubernetes.py

```diff
@@ -1,25 +1,88 @@
 from kubernetes_asyncio import client as k8s_client
 from kubernetes_asyncio.client.rest import ApiException as K8SApiException
 from ...exceptions.tardisexceptions import TardisError
-from ...interfaces.siteadapter import SiteAdapter
-from ...interfaces.siteadapter import ResourceStatus
+from ...interfaces.siteadapter import ResourceStatus, SiteAdapter, SiteAdapterBaseModel
 from ...utilities.attributedict import AttributeDict
 from ...utilities.staticmapping import StaticMapping
 from ...utilities.utils import convert_to
 
+from pydantic import AnyUrl, BaseModel, conint, constr, PositiveInt, validator
+
 from functools import partial
 from datetime import datetime
 from contextlib import contextmanager
+from typing import Any, List, Optional
 
 import logging
 
 logger = logging.getLogger("cobald.runtime.tardis.adapters.sites.kubernetes")
 
 
+class KubernetesMachineTypeConfigurationModel(BaseModel):
+    args: List[str]
+    cpu_utilization: Optional[conint(ge=0, le=100)]
+    hpa: bool = False
```

Comment:
Since I was anyhow working on the input parameters I took the opportunity make them a bit more reasonable. There is no need to always set `hpa=False` in the standard usage of this adapter. So, I made it default.